website/frontend/static/less/site-wide.less

```diff
@@ -21,24 +22,22 @@ a
 .navbar
 {
    margin: 0px;
-   border-width: 0 0 5px;
+   border-color: #fff;
    .navbar-brand
    {
       font-size: 20px;
    }
+   .nav{
+      border-bottom: 2px solid @mb-purple;
+   }
```

Comment:
You don't actually need to define these things yourself. They are part of a theme that we have for Bootstrap. :)

What you need to do is copy it into this project from, let's say, https://github.com/metabrainz/metabrainz.org/tree/master/metabrainz/static/css. After that all you need to do is modify *main.less*, which is the main styling file in case of MetaBrainz website. It looks like Picard website has multiple, but the process should be similar.